src/stepper/Stepper.tsx

```diff
@@ -34,6 +35,7 @@ interface StepperState {
 // Support IE. Same value returned with Number.MAX_SAFE_INTEGER / Number.MIN_SAFE_INTEGER
 const defaultInteger = 2 ** 53 - 1
 const isTouchScreen = isTouchEventsAvailable()
+const cssColorRegex = /^#([0-9a-f]{3}|[0-9a-f]{6})$/i
```

Comment:
We may need a "sanitize" helper to take care of this kind of verification. For our use case it's fine but maybe others will want to use rgb(a) formats for instance.

Reply:
Removed